README.md

```diff
@@ -82,6 +91,11 @@ describe "my server"
 end
 ```
 
+### Stubbing
+
+You can stub commands using `stub_command`. This function takes as first argument the name of the command you will stub. You can provide an optional argument that will be the body of the method. (code that would be evaluated)
```

Comment:
"This function takes the name of the command you wish to stub. If provided, the second argument will be used as the body of the command."

You should definitely link to the example tests [as originally discussed](https://github.com/rylnd/shpec/pull/49#discussion_r25998988).